scripts/text_control.js

```diff
@@ -0,0 +1,53 @@
+/*global $ */
+// this operates the font face, font size and wrap controls
+$(function () {
+    var fontFaceSelector = document.getElementById("fntFace");
```

Comment:
why jquery sometimes and other time just normal DOM APIs? For instance here could've been $('#...').

Reply:
No particular reason. Some of this was taken from old code. It would probably be better to be more consistent as you suggest.